web_accounts_list.json

```diff
@@ -4591,6 +4591,73 @@
            "known_accounts" : ["john", "junech"],
            "category" : "dating",
            "valid" : true
+          },
+          {
+           "name" : "icq-chat",
+           "check_uri" : "https://icq.icqchat.co/members/{account}/",
+           "account_existence_code" : "200",
+           "account_existence_string" : "ICQ chat",
+           "account_missing_string" : "Oops! We ran into some problems",
+           "account_missing_code" : "404",
+           "known_accounts" : ["brookenora.54", "bigdaddy.77"],
+           "category" : "social",
+           "valid" : true
+          },
+          {
+           "name" : "HeyLink.me",
+           "check_uri" : "https://heylink.me/{account}/",
+           "account_existence_code" : "200",
+           "account_existence_string" : "Heylink.me",

```

Comment:
I'm getting a bad detection string for this site. Can you research further? It might be that your tool is following a redirect instead of just looking at what is the first HTTP response

Reply:
Which tool are you using that is giving you a bad detection string return error?
I am using the python tool provided `web_accounts_list_checker.py`, this tool always detected a bad string, for the current string it's returning the below (see screenshot)
<br>
<img width="883"  src="https://user-images.githubusercontent.com/104733166/178166597-a5680194-d1a8-45cd-b766-fe813d3475bb.png">